.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,31 @@
+name: GstVkVideoParsers CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dependecies
+      run: |
+        sudo apt update
+        sudo apt install pkg-config
+        sudo apt install libglib2.0-dev libjson-glib-dev libncursesw5-dev
+        sudo apt install python3-pip python3-setuptools ninja-build
+        sudo apt install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev
+        sudo pip3 install --user meson
```

Comment:
I think there's no need for meson from pip... perhaps just from apt ... we'll  need to check

Reply:
I always prefer the pip3 one as it's more up2date. The ubuntu 22.04 version is 0.61, pip3 is 0.63.2 and minimum gstreamer version is 0.62